instruments/tests/test_tektronix/test_tekawg2000.py

```diff
@@ -241,14 +241,15 @@ def test_waveform_name_type_mismatch():
         assert exc_msg == "Waveform name must be specified as a string."
 
 
+@pytest.mark.skipif(numpy is None, reason="Numpy required for this test")
```

Comment:
Great use of `pytest.mark.skipif`, need to remember this for the future, this seems really useful for having optional dependencies!